modules/local/pacbio/hificnv/main.nf

```diff
@@ -2,14 +2,15 @@ process HIFICNV {
     tag "$meta.id"
     label 'process_medium'
 
-    conda "bioconda::hificnv=0.1.7"
-    container "quay.io/biocontainers/hificnv:0.1.7--h9ee0642_0"
-
+    conda "bioconda::hificnv=1.0.0"
+    container "quay.io/biocontainers/hificnv:1.0.0--h9ee0642_0"
```

Comment:
could you include the singularity option here as well: `https://depot.galaxyproject.org/singularity/hificnv:1.0.0--h9ee0642_0`